tests/test_cli.py

```diff
@@ -115,57 +114,60 @@ def test_command_list_node(workdir, mocker):
     ('traintuple', ['--objective-key', 'foo', '--algo-key', 'foo', '--dataset-key', 'foo',
                     '--data-samples-path']),
     ('testtuple', ['--traintuple-key', 'foo', '--data-samples-path'])]
-)
+                         )
```

Comment:
This change does not seem related to the PR.

Reply:
Yes, seems to be a mistake